src/parser.jl

```diff
@@ -130,7 +130,12 @@ end
 curline(ts::TokenStream)  = ts.lineno
 filename(ts::TokenStream) = symbol("none")
 
-line_number_node(ts) = LineNumberNode(curline(ts))
+if VERSION < v"0.4-dev+7034"
+    line_number_node(f, n) = LineNumberNode(n)
```

Comment:
why are you defining `line_number_node` with 2 arguments when it only gets used with one input?